x11-misc/ccsm/ccsm-0.8.14.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_IN_SOURCE_BUILD=1
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="A graphical manager for CompizConfig Plugin (libcompizconfig)"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk3"
+
+RDEPEND="
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	>=dev-python/compizconfig-python-0.8.12[${PYTHON_USEDEP}]
+	<dev-python/compizconfig-python-0.9
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	gnome-base/librsvg
+"
+
+python_prepare_all() {
+	# return error if wrong arguments passed to setup.py
+	sed -i -e 's/raise SystemExit/\0(1)/' setup.py || die 'sed on setup.py failed'
+
+	# correct gettext behavior
```

Comment:
Please remove this block. It's not strictly required, and really belong upstream.